packages/payment/src/models/payment-session.ts

```diff
@@ -43,6 +43,9 @@ export default class PaymentSession {
   @Property({ columnType: "jsonb" })
   data: Record<string, unknown> = {}
 
+  @Property({ columnType: "jsonb", nullable: true })
+  context?: Record<string, unknown> | null
```

Comment:
todo: rm ?

Reply:
I added the context, so we have the data for when we need to create a new payment session in the compensation flow of `deletePaymentSession`. The alternative would be to not have a compensation step, because the context is a required property for the creation at the third-party provider.

Reply:
Let me know what you think.

Reply:
I meant remove the `?` symbol 😂

Reply:
Ahh haha, I thought you were asking a question because of the `?` 😆